docs/core/gotchas.md

```diff
@@ -43,10 +43,10 @@ RuntimeError: cannot reshape tensor of 0 elements into shape [0, -1] because the
 The problem here is that no neighbors are found for the single atom which causes an error. This may be model dependent. There is currently no way to get atomic energies for some models.
 
 ```{code-cell} ipython3
-from fairchem.core.common.relaxation.ase_utils import OCPCalculator
+from fairchem.core.common.relaxation.ase_utils import FAIRChemCalculator
```

Comment:
Can you change all these imports to just be
```python
from fairchem.core import FAIRChemCalculator
```

Reply:
Thank you for the response. I will complete this as soon as I can.